summingbird-scalding/src/main/scala/com/twitter/summingbird/scalding/store/DirectoryBatchedStore.scala

```diff
@@ -81,7 +81,7 @@ class DirectoryBatchedStore[K <: Writable, V <: Writable](val rootPath: String)(
 
         val lastBatchStatus =
           hdfsPaths.map(getFileStatus(_, conf))
-            .filter { input => input._1 && (BatchID(input._2) < exclusiveUB) }
+            .filter { input => input._1 && (BatchID(input._2.milliSinceEpoch) < exclusiveUB) }
```

Comment:
Looks like @reconditesea wrote this code. It seems like it should be `batcher.batchOf(input._2)` and while we are at it, I'd rewrite this as:

```scala
.filter { case (isGood, timestamp, _) => isGood && (batcher.batchOf(timestamp) < exclusiveUB) }
```
I guess there are no tests on this code. :/